src/procedures/procedures.11tydata.yaml

```diff
@@ -0,0 +1,284 @@
+          <li class="info">Zigbee2MQTT uses the MQTT discovery feature to make the Zigbee devices available in Home Assistant. After removing the Zigbee2MQTT add-on, the devices are still present in Home Assistant since the MQTT broker is still holding on to the MQTT discovery messages.</li>
+          <li class="info">It is recommended to remove these stale devices from the MQTT broker (see next steps).</li>
+        </ul>
+    - title: Remove Zigbee2MQTT devices (option 1)
+      image: z2m-mqq-integration-01.png
+      content: |
+        <ul>
+          <li>If you don't use MQTT outside z2m, you can delete the Mosquito Broker integration. Under <b>Devices & Services</b>, in the <b>Mosquitto broker</b> integration, select <b>Delete</b>.</li>
+        </ul>
+    - title: Remove Zigbee2MQTT devices (option 2)
+      image: z2m-mqtt-dereg-device-01.png
+      content: |
+        <ol>
+          <li class="info">If you don't want to delete the Mosquito Broker integration, you can remove the devices from the MQTT network.</li>
+          <li>In the integration card, select <b>Devices</b>, under <b>Device info</b>, select <b>Delete</b>.</li>
+          <li>Repeat this for every device.</li>
+        </ol>
+    - title: Plug the extension cable into SkyConnect
+      image: skyconnect-raspi-01.jpg
+      content: |
+        <ol>
+          <li>Plug the Home Assistant SkyConnect into the USB extension cable.</li>
+          <li>Plug the extension cable into your Home Assistant instance.</li>
+          <li class="warning">It is important to use the USB extension cable. USB 3.0 ports and USB 3.0 drives can cause interference with 2.4 GHz protocols, including the ones provided by Home Assistant SkyConnect.</li>
+        </ol>
+    - title: Locate the SkyConnect integration
+      image: skyconnect-discovered-01.png
+      content: |
+        <ol>
+          <li>Go to <b>Settings</b>><b>Devices & Services</b>.</li>
+          <li>SkyConnect should now have been discovered.</li>
+          <li>Select <b>Configure</b>.</li>
+        </ol>
+    - title: Upload a manual backup
+      image: z2m-backup-04.png
+      content: |
+        <ul>
+          <li>In the <b>Network Formation</b> dialog,  select <b>Upload a manual backup</b>.</li>
+        </ul>
+    - title: Select the backup file
+      image: z2m-backup-restore-02.png
+      content: |
+        <ul>
+          <li>In your <b>Downloads</b> folder, select the <b>coordinator_backup.json</b> file.</li>
+        </ul>
+    - title: Submit the backup json file
+      image: z2m-backup-restore-03.png
+      content: |
+        <ul>
+          <li>Make sure the <b>coordinator_backup.json</b> file is selected and select <b>submit</b>.</li>
+        </ul>
+    - title: Assign devices to area
+      image: z2m-backup-restore-04.png
+      content: |
+        <ul>
+          <li>Assign each of the devices to an area and select <b>Finish</b>.</li>
```

Comment:
Devices appearing during ZHA setup is an artifact of the testing setup: `/config/zigbee.db` persists even if you delete ZHA, so devices you previously added will be picked up again when you re-add ZHA.  If you delete the file before setting up ZHA, no devices other than the coordinator will show up, which is what a person will typically see if they haven't used ZHA before.

Reply:
implemented.